src/sagemaker/serve/builder/model_builder.py

```diff
@@ -881,8 +881,8 @@ def _build_for_model_server(self):  # pylint: disable=R0911, R1710
         if self.model_metadata:
             mlflow_path = self.model_metadata.get(MLFLOW_MODEL_PATH)
 
-        if not self.model and not mlflow_path:
-            raise ValueError("Missing required parameter `model` or 'ml_flow' path")
+        if not self.model and not mlflow_path and not self.inference_spec:
```

Comment:
Do we need this here? Since we already have a PR for it? https://github.com/aws/sagemaker-python-sdk/pull/4769

Reply:
yes these changes can be remove, since i've moved this to another PR and added UT coverage for it

Reply:
Did the sync, thank you